addons/binding/org.openhab.binding.regoheatpump/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,721 @@
+    </channel-type>
+
+    <channel-type id="adjCurveAtMinus30" advanced="true">
+        <item-type>Number</item-type>
+        <label>Adjust heat curve at -30 °C</label>
+        <description>Heat curve can be adjusted up or down every fifth outdoor degree. The purpose of breaking the curve is to be able to influence the heat pump's heat production at extra sensitive outdoor temperatures</description>
+        <category>Temperature</category>
+        <state readOnly="true" pattern="%.1f °C"/>
+    </channel-type>
+
+    <channel-type id="adjCurveAtMinus35" advanced="true">
+        <item-type>Number</item-type>
+        <label>Adjust heat curve at -35 °C</label>
+        <description>Heat curve can be adjusted up or down every fifth outdoor degree. The purpose of breaking the curve is to be able to influence the heat pump's heat production at extra sensitive outdoor temperatures</description>
+        <category>Temperature</category>
+        <state readOnly="true" pattern="%.1f °C"/>
+    </channel-type>
+
+    <channel-type id="heatCurveCouplingDiff" advanced="true">
+        <item-type>Number</item-type>
+        <label>Heat curve coupling diff</label>
+        <category>Temperature</category>
+        <state readOnly="true" pattern="%.1f °C"/>
+    </channel-type>
+    
+    <channel-type id="airIntakeTemp" advanced="true">
+        <item-type>Number</item-type>
+        <label>Air intake</label>
+        <category>Temperature</category>
+        <state readOnly="true" pattern="%.1f °C"/>
+    </channel-type>
+    
+    <channel-type id="poolTemp" advanced="true">
+        <item-type>Number</item-type>
+        <label>Pool</label>
+        <category>Temperature</category>
+        <state readOnly="true" pattern="%.1f °C"/>
+    </channel-type>
+
+    <channel-type id="compressorSpeed" advanced="true">
+        <item-type>Number</item-type>
+        <label>Compressor speed</label>
+        <category>Pump</category>
+        <state readOnly="true" pattern="%.1f %%"/>
+    </channel-type>
+
+    <channel-type id="heatCurve2" advanced="true">
+        <item-type>Number</item-type>
+        <label>Heat curve 2</label>
+        <category>Line</category>
+        <state readOnly="true" pattern="%.1f °C"/>
+    </channel-type>
+        
+    <channel-type id="switchValve2State" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Switch valve 2</label>
+        <category>Heating</category>
+        <state readOnly="true" pattern="%s"/>
+    </channel-type>
+    
+    <channel-type id="fanState" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Fan</label>
+        <category>Fan</category>
+        <state readOnly="true" pattern="%s"/>
+    </channel-type>
+
+    <channel-type id="highPressostatState" advanced="true">
+        <item-type>Switch</item-type>
+        <label>High pressostat</label>
+        <category>Switch</category>
+        <state readOnly="true" pattern="%s"/>
```

Comment:
I removed the pattern, but remembered why I added them in the first place - please see [screenshot](https://community-openhab-org.s3-eu-central-1.amazonaws.com/original/2X/7/7d8a05290855e38ebc03febea39445775258eba9.jpg) from the previous version. If you check the switch items under the front panel section, there is a text on/off on the right side. Now that I removed the pattern, the text is not there anymore... Is this a bug or a feature? :)